test/addons/async-hello-world/binding.cc

```diff
@@ -13,7 +20,8 @@ struct async_req {
 
 void DoAsync(uv_work_t* r) {
   async_req* req = reinterpret_cast<async_req*>(r->data);
-  sleep(1);  // Simulate CPU intensive process...
+  // Simulate CPU intensive process...
+  std::this_thread::sleep_for(std::chrono::seconds(1));
```

Comment:
But since the latest v8 is also using c++11, I think OS X might be able to get around c++11 problems by using new compilers and libraries.